### PR TITLE
Feat: addons/routes

### PR DESCRIPTION
**Pull request by oxedions**
Role to define routes.

### CODE REVIEW

**Comment by oxedions** [comment]
Forgot to set version in vars.

File: resources/packaging/bluebanquise.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %{!?version: %define version 1.3.0}
 
 %define roles_addons clone clustershell diskless grafana lmod nic_nmcli ofed \
-ofed_sm openldap_client openldap_server powerman prometheus_client \
+ofed_sm openldap_client openldap_server powerman prometheus_client routes \
```
Comment (btravouillon) [inline resources/packaging/bluebanquise.spec:4]: Can you keep the list sorted alphabetically?

File: roles/addons/routes/readme.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              mac: 08:00:27:36:c0:ac
+              network: ice1-1
+              routes4:
+                - 10.11.0.0/24 10.10.0.2
+                - 10.12.0.0/24 10.10.0.2
+
+.. note::
+  Note that to define a default route/gateway, use *0.0.0.0/0* as route to be defined.
+
+To remove a route later (here *10.12.0.0/24 10.10.0.2* on *enp0s8*), use the nmcli command this way:
```
Comment (btravouillon) [inline roles/addons/routes/readme.rst:47]: The role must provide a way to remove a given route.

File: roles/addons/routes/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+
```
Comment (btravouillon) [inline roles/addons/routes/tasks/main.yml:2]: Please remove the blank line.

**Comment by oxedions** [comment]
Hello.
I am canceling this PR. New plans for it, it will be integrated to nic_nmcli.

**Pull request closed without merging**